### ISSUE TITLE
Add a sandbox version env var

### ISSUE THREAD

**Issue by reconbot** [architect/architect#1269]
I'd like to know if my code is running in the sandbox that supports new features, such as the websocket management api. If the sandbox is too old I can't support all the features and would like to tell the user why. 

I'd like to expose the current sandbox version in `ARC_SANDBOX_VERSION`.

We can try to figure out what version of the sandbox is by poking it with http requests but that seams fraught.

This came up working on https://github.com/architect/functions/pull/499 we'd like to support `send()` on all versions of the sandbox and the rest of the websocket functions on the supporting versions of the sandbox (`@architect/sandbox@4.3.0` or higher) however the implementation of `send` is different for the newer sandbox.

**Comment by ryanblock** [comment]
I’m thinking a small json serialized object that includes Sandbox version, maybe API type, and perhaps some other metadata if we can think if anything else. Or just version and nothing else!

**Comment by ryanblock** [comment]
Shipped in Sandbox 4.3.1 + Arc 9.3.1 with the following shape:
```
JSON.parse(process.env.ARC_SANDBOX)
// {
//   apiType: 'http' | 'httpv1' | 'rest',
//   cwd: '/path/to/your/project',
//   ports: {
//     httpPort: 3333,   // default
//     eventsPort: 3334, // default
//     tablesPort: 5000, // default
//     _arcPort: 3332,   // default
//   },
//   staticPath: '/path/to/your/project/static-asset-dir',
//   version: '4.3.1',
// }
```
